src/NzbDrone.Core/Localization/Core/en.json

```
@@ -485,6 +485,8 @@
   "DownloadClientQbittorrentSettingsContentLayoutHelpText": "Whether to use qBittorrent's configured content layout, the original layout from the torrent or always create a subfolder (qBittorrent 4.3.2+)",
   "DownloadClientQbittorrentSettingsFirstAndLastFirst": "First and Last First",
   "DownloadClientQbittorrentSettingsFirstAndLastFirstHelpText": "Download first and last pieces first (qBittorrent 4.1.0+)",
+  "DownloadClientQbittorrentSettingsPropagateTags": "Propagate Tags",
+  "DownloadClientQbittorrentSettingsPropagateTagsHelpText": "Add Season Tags to new torrents when added to the download client (qBittorrent 4.1.0+)",
```

In the context of Sonarr, series = show, not series = season so it'd be:

```suggestion
  "DownloadClientQbittorrentSettingsPropagateTagsHelpText": "Add series tags to new torrents when added to the download client (qBittorrent 4.1.0+)",
```